doc/whats_new.rst

```diff
@@ -8,7 +8,7 @@ These are new features and improvements of note in each release
     :local:
     :backlinks: top
 
-.. include::  whatsnew/v0-4-0.rst
+.. include::  whatsnew/v0-3-3.rst
```

Comment:
In dev, we should have both whatsnews.

Reply:
As I understood, we will skip v0.3.3, won't we?

Reply:
In that case, I would just rename the file v0.3.3 -> v0.4.0